### DISCUSSION TITLE
PCF8574 with script

### DISCUSSION THREAD

**Discussion by Bitfriemler22** [arendst/Tasmota#15437] [Q&A] [answered]
Hi there,

I use the PCF8574 as port extender which works fine. from GUI
Now I would like to set outputs using script language.
Unfortuantelly I cant't figure out which command I should use.
Using iw(AA val) this works successfully but doesn't update GUI and maybe other Tasmota internals.
Any hint which command I shall use is very welcome!
Thanks

**Comment by barbudor** [comment, marked as answer]
Normally PCF outputs are considered as Relay so should be controllable through Power command

**Comment by Bitfriemler22** [comment]
Thanks barbudor!
This is what I expected and documentation confirms that pins can be used with "Power" https://tasmota.github.io/docs/PCF8574/#tasmota-configuration.
From the command line I tried "script >Power4=1" but no change in GUI.
Using a logic analyser there is still &FF read back (I have inverted logic).
Am I using wrong syntax perhaps?

**Reply by barbudor** [reply]
I have no experience with script but a syntax that works with a GPIO defined as  Relay should work also with a PCF output
In the console directly typing 'Power on' or 'Power off' should work

**Reply by sfromis** [reply]
Do not expect automatic web UI updating, only rely on what you see there after doing a refresh. Much more reliable to read what you see in the console.

**Reply by gemu2015** [reply]
from script Tasmota cmds are executed via =>

so use e.g. =>power1 on

**Reply by Bitfriemler22** [reply]
The hint from gemu2015 worked for me, thanks a lot you all.
